test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+  test("returns a reversed array of the number's digits", () => {
+    expect(numberToReversedDigits(12345)).toEqual([5, 4, 3, 2, 1]);
```

Comment:
Would be good to have some more assertions here, what about for single numbers or much longer ones?